src/Language/Fixpoint/Types/Names.hs

```diff
@@ -44,6 +44,7 @@ module Language.Fixpoint.Types.Names (
   , dropSym
   , headSym
   , lengthSym
+  , mappendSym
```

Comment:
Yes, this code https://github.com/ucsd-progsys/liquidhaskell/blob/f399e197dec7bc82575a2d2ce6be34bbc06283c4/benchmarks/haskell16/pos/Proves.hs

creates dictionaries 

`$fOptEqa(->)` and `$fOptEqa(->)0`